commit cbc43816a223011285f933b604cc79049488ddcf (refs/changes/20/2420/3)
Author: Richard Bradley <richard.bradley@joyent.com>
Date:   2017-08-17T19:24:04+01:00 (2 years, 2 months ago)
    
    MANATEE-352 Add support for "synchronous_commit=off" in pg_overrides
    Reviewed by: Sean Chittenden <sean.chittenden@joyent.com>
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/package.json b/package.json
index aa1e0b2..9bbd8a1 100644
--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     },
     "dependencies": {
         "manta": "1.2.6",
-        "manatee": "git+https://github.com/joyent/manatee.git#45f8cbd2",
+        "manatee": "git+https://github.com/joyent/manatee.git#34238c25",
         "carrier": "0.2.0"
     },
     "sdcDependencies": {
